tests/testthat/test_comp_analysis.R

```diff
@@ -7,7 +7,7 @@ library(geiger)
 test_that("We can extract tree and trait data to run fitContinuous and fitDiscrete", {
   nexml <- read.nexml(system.file("examples", "comp_analysis.xml", package="RNeXML"))
   traits <- get_characters(nexml)
-  tree <- get_trees(nexml)
+  tree <- get_trees(nexml) 
```

Comment:
That's an unrelated whitespace-only change, right?

Reply:
whoops, yes,I  shouldn't be introducing random whitespace changes